src/Functions/iterable_to_array.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace JMS\Serializer\Functions;
+
+use function array_values;
+use function iterator_to_array;
+
+/**
+ * Copy the iterable into an array. If the iterable is already an array, return it.
+ *
+ * @param bool $use_keys [optional] Whether to use the iterator element keys as index.
+ *
+ * @return array
+ */
+function iterable_to_array(iterable $iterable, bool $use_keys = true) : array
```

Comment:
`$use_keys` looks never used in the calling code

Reply:
Copied it from library I use so I don't have to include it whole https://github.com/bpolaszek/php-iterable-functions/blob/master/src/iterable-functions.php#L34

I removed `$use_keys`